src/components/textfield/textfield.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components';
+import { css } from 'styled-components';
+
+const TextXS = css`
+  font-size: 12px;
+  line-height: 150%;
+`;
+
+const TextR = css`
+  font-size: 16px;
+  line-height: 150%;
+`;
```

Comment:
To differentiate between styled components and mere snippets of CSS I suggest to start these snippets in lower case, like regular variables. I know that in the past I might have suggested `TextR`, but reading this now I think it might get confusing in the future. So my suggestion would be to use `textXS` and `textR` instead of `TextXS` and `TextR`. This suggestion is backed by a comment in the styled-components repository: https://github.com/styled-components/styled-components/issues/773#issuecomment-299667585

I know that you used the "upper camel case" naming schema in the Headerblock PR already and I'm fine with both variants. This is obviously not a blocker and we can easily refactor this later. Maybe you have an opinion on this and prefer the one or the other. Feel free to keep the names as they are or rename them if you agree.

Reply:
Yes, I like the lower Case schema for those css snippets